doc/migrate-to-pymongo4.rst

```diff
@@ -50,6 +50,12 @@ Warnings can also be changed to errors::
 .. note:: Not all deprecated features raise :exc:`DeprecationWarning` when
   used. See `Removed features with no migration path`_.
 
+MongoReplicaSetClient
+---------------------
+
+Removed :class:`~pymongo.mongo_replica_set_client.MongoReplicaSetClient`.
+Use :class:`pymongo.mongo_client.MongoClient` instead.
```

Comment:
I'd suggest mentioning that since PyMongo 3.0 MongoReplicaSetClient has been a trivial wrapper for MongoClient that only overrode __repr__, to make it clear that the migration literally only requires a change of class name.

Reply:
Done. I didn't mention the repr though.